app/jobs/spotlight/add_uploads_from_csv.rb

```diff
@@ -19,5 +20,13 @@ def perform(csv_data, exhibit, user)
       end
 
     end
+    private
+    def encoded_csv(csv)
+      csv.map do |row|
+        row.map do |label, column|
+          [label, column.encode('UTF-8', invalid: :replace, undef: :replace, replace: "?")] if column.present?
```

Comment:
Replaced w/ `\uFFFD`.  I tried String#scrub (under ruby 2.1) but it seemed to still throw the same error.